modules/ldap-auto-syncing.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-auto-syncing_{context}"]
+= Automatically syncing LDAP groups
+
+You can automatically sync LDAP groups on a periodic basis by configuring a cron job.
```

Comment:
Nit: maybe a link to cron job page?

Reply:
I do have a link to the cron job page below this syncing section in an "Additional resources" section. It's a workaround because we unfortunately can't link to other sections in the docs in modules right now :disappointed: